x-pack/plugins/secops/public/components/page/network/network_top_n_flow_table/columns.tsx

```diff
@@ -171,39 +163,54 @@ export const getNetworkTopNFlowColumns = (
               expression={`network.direction: ${escapeQueryValue(direction)}`}
               type={type}
             >
-              {defaultToEmptyTag(direction)}
+              <>
+                {defaultToEmptyTag(direction)}
+                {index < directions.length - 1 ? '\u00A0' : null}
```

Comment:
Isn't `&nbsp;` what most people use for non-breaking space? I am assuming it doesn't work here?

Reply:
yes, it did not work, so I used the Unicode way